magicbyirineu/Persistence/Repository/RealmCards.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+class RealmCards {
```

Comment:
Esse nome não está muito bom.

```suggestion
class DatabaseCardsService {
```

Reply:
Refatorado!